internal/infrastructure/event/factory.go

```diff
@@ -22,13 +22,13 @@ func Start(ctx context.Context, config *config.Kafka, handler Eventable) {
 	)
 
 	if consumer, err = NewConsumer(config); err != nil {
-		log.Panic().Msgf("error creating consumer: %s", err.Error())
+		slog.Error("error creating consumer", err.Error())
```

Comment:
```suggestion
		slog.Error("error creating consumer", slog.Any("error", err))
```

Reply:
Fixed, thanks

Reply:
good catch! I have reviewed and the last update include this change! resolving the conversation.